pkg/engagement/usecases/fcm/push.go

```diff
@@ -0,0 +1,39 @@
+package fcm
+
+import (
+	"context"
+
+	"github.com/savannahghi/engagement/pkg/engagement/infrastructure"
+	"github.com/savannahghi/firebasetools"
+)
+
+// PushUsecase defines FCM push usecases interface
+type PushUsecase interface {
+	PushFn(
```

Comment:
Renaming things. `push` - `pushFn` might cause some interesting problems. Avoid it if it's not necessary.